types/valuerenderer/valuerenderer.go

```diff
@@ -14,6 +14,8 @@ import (
 	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
 )
 
+var RegExp = regexp.MustCompile(`(\d+)(\w+)`)
```

Comment:
I have made 'RegExp` global variable since `valuerenderer_test` package imports that. Does it sound good ?